readme.md

```diff
@@ -129,9 +129,9 @@ The only use-case I can think of is having the config located in the app directo
 Type: `string` `Buffer` `TypedArray` `DataView`<br>
 Default: `undefined`
 
-Note that this is **not intended for security purposes**, since the encryption key would be easily found inside a plain-text Node.js app.
+This can be used to secure sensitive data **if** the encryption key is stored in a secure manner (not plain-text) in the Node.js app.
```

Comment:
I think you should write a little bit more on how to store the encryption key in a secure manner. 

You mentioned some of this in the PR comments:

> If the key used to decrypt the data is stored with keytar or is a password entered by the user (stored in a variable by example), this is secure.